### PR TITLE
[WNMGDS-1580] Use i18n for components A-F

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

See https://github.com/CMSgov/design-system/pull/1756

## How to test

`yarn test:unit`

### CODE REVIEW

File: packages/design-system/src/components/FilterChip/FilterChip.tsx [hunk 1]
```diff
@@ -81,6 +78,9 @@ export class FilterChip extends React.Component<FilterChipProps> {
         className={buttonClassNames}
         onClick={this.handleClick}
         onKeyDown={this.handleKeyDown}
+        aria-label={`${label} . ${
+          ariaClearLabel ?? t('filterChip.ariaClearLabel')
+        } ${label} filter`}
```
Comment (scheul93) [inline packages/design-system/src/components/FilterChip/FilterChip.tsx:83]: does the word 'filter' need to be localized too?

Comment (pwolfert) [reply packages/design-system/src/components/FilterChip/FilterChip.tsx:83]: Ah, yep, good catch

Comment (pwolfert) [reply packages/design-system/src/components/FilterChip/FilterChip.tsx:83]: The word order might even be different in Spanish for the phrase "_something_ filter"

Comment (pwolfert) [reply packages/design-system/src/components/FilterChip/FilterChip.tsx]: Got translations for this and updated the code

**Review by scheul93 — APPROVED** [review]

**Pull request merged**